README.md

```diff
@@ -55,10 +55,10 @@ jobs:
           fetch-depth: 0
           token: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Set up Node.js 16.x
+      - name: Set up Node.js
         uses: actions/setup-node@v3
```

Comment:
How about also bumping our own actions so we don't get the warning?

Reply:
https://github.com/seek-oss/changesets-snapshot/pull/12/commits/f1e687ef8ab91d3360c8956c36fd971b6e345a23